🐛 CI failure: tests/unit_tests/elastification/test_loss_func.py — ImportError on unwrap_model

**Describe the bug**

CI test `tests/unit_tests/elastification/test_loss_func.py` failed to collect in job [`tests/unit_tests/**/*.py - latest`](https://github.com/NVIDIA/Megatron-LM/actions/runs/26394060045/job/77690797642). The test module imports `loss_func` from `megatron.elastification`, which in turn imports `unwrap_model` from `megatron.training.utils` — that symbol no longer exists in `megatron/training/utils/__init__.py`.

Tag @NVIDIA/mcore-oncall to get oncall's attention to this issue.

**Failing run**

| Field | Value |
|-------|-------|
| PR    | [#4931: test: enable NVTE_CUTEDSL_FUSED_GROUPED_MLP via pytest fixture](https://github.com/NVIDIA/Megatron-LM/pull/4931) |
| Run   | [26394060045](https://github.com/NVIDIA/Megatron-LM/actions/runs/26394060045) |
| Job   | [tests/unit_tests/**/*.py - latest](https://github.com/NVIDIA/Megatron-LM/actions/runs/26394060045/job/77690797642) |

**Error**

```
==================================== ERRORS ====================================
______ ERROR collecting tests/unit_tests/elastification/test_loss_func.py ______
ImportError while importing test module '/opt/megatron-lm/tests/unit_tests/elastification/test_loss_func.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/unit_tests/elastification/test_loss_func.py:17: in <module>
    from megatron.elastification import loss_func as loss_func_module
megatron/elastification/loss_func.py:14: in <module>
    from megatron.training.utils import unwrap_model
E   ImportError: cannot import name 'unwrap_model' from 'megatron.training.utils' (/opt/megatron-lm/megatron/training/utils/__init__.py)
```

**Steps/Code to reproduce bug**

Re-run the failing CI job linked above, or locally inside the dev container:

```bash
pytest tests/unit_tests/elastification/test_loss_func.py
```

**Additional context**

Triaged automatically via `/create-issue`. The full log is available via the job URL.

--- a/megatron/elastification/loss_func.py
+++ b/megatron/elastification/loss_func.py
@@ -10,8 +10,8 @@ import torch
 
 from megatron.core import parallel_state
 from megatron.core.models.gpt import GPTModel
+from megatron.core.utils import unwrap_model
 from megatron.training import get_args
-from megatron.training.utils import unwrap_model
 
 
 def _mask_loss(output_tensor, loss_mask):

--- a/tests/unit_tests/elastification/test_loss_func.py
+++ b/tests/unit_tests/elastification/test_loss_func.py
@@ -16,11 +16,8 @@ import torch
 
 pytestmark = pytest.mark.flaky_in_dev
 
-try:
-    from megatron.elastification import loss_func as loss_func_module
-    from megatron.elastification.loss_func import _mask_loss, loss_func
-except ImportError as exc:
-    pytest.skip(f"megatron.elastification.loss_func unavailable: {exc}", allow_module_level=True)
+from megatron.elastification import loss_func as loss_func_module
+from megatron.elastification.loss_func import _mask_loss, loss_func
 
 
 def _stub_args(**overrides):

